config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -198,3 +198,99 @@ presubmits:
       containers:
       - image: gcr.io/kubeflow-ci/test-worker:latest
         imagePullPolicy: Always
+
+  kubeflow/pipelines:
+  - name: build-image
+    always_run: true
+    skip_report: false
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
```

Comment:
ref https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml#L11-L16

so this can become 
```
  - name: build-image
    always_run: true
    decorate: true
    labels:
      preset-service-account: "true"
    spec:
      containers:
      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
        command:
        - ./test/presubmit-tests.sh
        args:
        - --workflow_file
        - build_image.yaml
```

Reply:
(`skip_report` is also have a default false so we can drop that)

Reply:
done